ioet_feature_flag/strategies/pilot_users.py

```diff
@@ -0,0 +1,25 @@
+import typing
+
+from .strategy import Strategy
+from ..exceptions import MissingToggleAttributes
+from ..toggle_context import ToggleContext
+
+
+class PilotUsers(Strategy):
+    def __init__(self, enabled: bool, allowed_users: typing.List[str]) -> None:
+        self._enabled = enabled
+        self._allowed_users = allowed_users
+
+    @classmethod
+    def from_attributes(cls, attributes: typing.Dict) -> "PilotUsers":
+        allowed_users = attributes.get('allowed_users')
+        if not allowed_users:
+            raise MissingToggleAttributes("You must provide a list of allowed users")
+
+        return cls(
+            enabled=attributes.get('enabled', False),
```

Comment:
Quick question: What do you think of defaultig the enabled attribute to `True` if it is not present on the attribute dictionary? Reasoning here is that my understanding of this strategy is to enable a group of users to access a new functionality

Reply:
Oh I was thinking of defaulting them to `False` in case an user wants to initialize a feature but they want to enable it later, or as a safety measure to prevent someone enabling the feature on accident, what do you think?